package.json

```diff
@@ -4,11 +4,9 @@
   "description": "Serializer according to the accept header",
   "main": "index.js",
   "scripts": {
-    "lint": "standard | snazzy",
-    "unit": "mocha --check-leaks --full-trace --throw-deprecation --allow-uncaught test/*.js",
-    "test": "npm run lint && npm run unit",
-    "coverage": "nyc --reporter=lcov --reporter=html --check-coverage --lines 100 npm run unit",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls"
```

Comment:
this is needed by CI

Reply:
It can work without that actually. tap can automatically forward to coveralls. 

https://github.com/cemremengu/fastify-oracle/blob/master/package.json#L6

Let me see if that works